[DOCS] Generating secure user passwords for EMT builds (#332)
open-edge-platform/edge-microvisor-toolkit: docs/developer-guide/get-started/emt-building-howto.md

   make build-packages # to rebuild the packages
   ```

2. Build the image containing the package by following the steps outlined in [Building the Edge Microvisor Toolkit Image](#build-the-edge-microvisor-toolkit-image), and pointing to your modified imageconfig file.

### Example 3: Generating user passwords

To specify user passwords for your build, you can use

> [OpenSSL](https://openssl-library.org/source/) and generate a SHA-512 hash along with a salt,

Reviewer: ```suggestion
[OpenSSL](https://openssl-library.org/source/) and generate a SHA-512 hash with a random salt.
```

Author: > 

```suggestion
[OpenSSL](https://openssl-library.org/source/) and generate a SHA-512 hash with salt.
```

> which is the default and recommended security solution in the toolkit.

Reviewer: ```suggestion
Using SHA-512 hash with salt is the default and recommended security solution in the toolkit.
```

Author: > 

```suggestion
It is the default and recommended security solution in the toolkit.
```

1. Generate a secure password with `openssl passwd`:

   ```bash
   openssl passwd -6 -salt YourSaltValue UserPassword
   ```

   The `-6` parameter is required for creation of a secure SHA-512 hash. `-salt` is optional,

>    but recommended, as it provides a more complex and unique password.

Reviewer: ```suggestion
   but recommended. It adds random characters, providing a more complex and unique password.
```
